src/main/resources/org/jenkinsci/plugins/edithelp/EditHelpRootAction/get.jelly

```diff
@@ -0,0 +1,4 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    ${it.myString}
```

Comment:
🐛 IIRC you were going to support HTML-formatted help. The variable is escaped by default, hence all HTML syntax will be escaped as well. Use <code><j:out></code> tag